lang/python/python-package.mk

```diff
@@ -68,8 +109,12 @@ define PyPackage
   $(call shexport,PyPackage/$(1)/filespec)
 
   define Package/$(1)/install
+	$(if $(strip $(filter "$(PyBuild/Compile/Default)","$(PyBuild/Compile)")),\
+		$$(call Build/Setup/PyMod,$$(PYTHON_PKG_install_SUBDIR), \
+		$$(PYTHON_PKG_install_ARGS), \
+		$$(PYTHON_PKG_install_VARS) \
+	))
 	$(call PyPackage/$(1)/install,$$(1))
-	find $(PKG_INSTALL_DIR) -name "*\.exe" | xargs rm -f
```

Comment:
Need to double check that python-package-install.sh takes of this.  If not it was a slip of the finger.

Reply:
fixed